suite/tests/api/ir_x86_0args.h

```diff
@@ -73,6 +73,8 @@ OPCODE(fxam, fxam, fxam, 0)
 OPCODE(sahf, sahf, sahf, 0)
 OPCODE(mwait, mwait, mwait, 0)
 OPCODE(monitor, monitor, monitor, 0)
+OPCODE(mwaitx, mwaitx, mwaitx, 0)
+OPCODE(monitorx, monitorx, monitorx, 0)
```

Comment:
Please also enable the binutils tests for these, which are currently disabled:

```
$ git grep -E 'mwaitx|monitorx'
third_party/binutils/test_decenc/test_decenc_x86.asm:        /* FIXME i3577: monitorx, AMD only. */
third_party/binutils/test_decenc/test_decenc_x86.asm:        /* FIXME i3577: mwaitx, AMD only. */
third_party/binutils/test_decenc/test_decenc_x86_64.asm:        /* FIXME i3577: monitorx, AMD only. */
third_party/binutils/test_decenc/test_decenc_x86_64.asm:        /* FIXME i3577: mwaitx, AMD only. */
```

Reply:
this one was very tricky and I had to reorder tests in binutils due to jxx/call + rel32; done though

Reply:
That is the downside of golden output tests, though making all these binutils tests into internal separate checks would be even more work.  I recall having to take the new full output on an update and verifying the final diff was all rel jmps: not too bad.